src/style.css

```diff
@@ -1 +1,103 @@
 /* Styles go here */
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+  font-family: "Roboto", sans-serif;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+}
+
+.header {
+  width: 1200px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  box-sizing: border-box;
```

Comment:
move to the `*` selector
```suggestion
```

Reply:
why you ignore that?

Reply:
> why you ignore that?

First time, I tried to do it after your review. At that time I had 2 headers in my code. It broke down everything. So, I just removed it completely, since it actually did nothing, it was a leftover. 
 
After the second review, when I was advised to remove 50 px margins from nav and homebutton and add padding 0 50px  for header, I started to need it again. So I added it again, but forgot about moving it to * and didn't even try to do it.

PS
In the 3rd review, I was advised to leave only one header (I did not get from the task that only 1 header was needed). That was what helped me to pass the autotests.